integration_test/workflow_tests/test_packages/ingest/test_multiple_granules_happy_path.py

```diff
@@ -27,19 +28,29 @@ def test_multiple_granules_happy_path(self):
             createdAt_time = int((time.time() + 5) * 1000)
             collection_name = uuid.uuid4().__str__()
             collection_version = uuid.uuid4().__str__()
-            # standard bucket where initial file exists
+            # standard bucket where test files will be copied
             cumulus_bucket_name = "orca-sandbox-s3-provider"
             recovery_bucket_name = helpers.recovery_bucket_name
             excluded_filetype = []
-
-            name_1 = "ancillary_data_input_forcing_ECCO_V4r4.tar.gz"
-            key_name_1 = "PODAAC/SWOT/" + name_1
+            name_1 = uuid.uuid4().__str__() + ".hdf"    # refers to file1.hdf
+            key_name_1 = uuid.uuid4().__str__() + "/" + name_1 
             file_1_hash = uuid.uuid4().__str__()
             file_1_hash_type = uuid.uuid4().__str__()
-            name_2 = "MOD09GQ.A2017025.h21v00.006.2017034065104_ndvi.jpg"
-            key_name_2 = "MOD09GQ/006/" + name_2
+            name_2 = uuid.uuid4().__str__()+ ".tif"  # refers to file2.tif
+            key_name_2 = uuid.uuid4().__str__() + "/" + name_2
             execution_id = uuid.uuid4().__str__()
 
+            # Upload the randomized file to source bucket
+            try:
```

Comment:
Exceptions are raised automatically. You can remove the `try... except...` blocks with no functional change.

Reply:
removed